### PR TITLE
AP-568 set up es-indexer Customer Prop

### PR DESCRIPTION
**Pull request by dan-dawson**

### CODE REVIEW

**Review by MarcinGinszt — APPROVED** [review]

File: dev-aws/kafka-shared-msk/customer-proposition/customer-proposition.tf [hunk 1]
```diff
@@ -85,3 +85,15 @@ module "uswitch-reporter" {
   produce_topics   = [kafka_topic.uswitch_events_v1.name]
   cert_common_name = "customer-proposition/uswitch-reporter"
 }
+
+module "es-indexer-uswitch-data-v1" {
```
Comment (mwindlinger) [inline dev-aws/kafka-shared-msk/customer-proposition/customer-proposition.tf:89]: you need to specify the consumer-group as well, like we did here https://github.com/utilitywarehouse/kafka-cluster-config/blob/bce437d40fad07523bb6741fea066f4a74129150/dev-aws/kafka-shared-msk/customer-proposition/customer-proposition.tf#L52

Comment (dan-dawson) [reply dev-aws/kafka-shared-msk/customer-proposition/customer-proposition.tf:89]: ah ok. So in this case would it be `es-indexer-uswitch.events.v1`? or does in need the namespace prefix as well?

Comment (mwindlinger) [reply dev-aws/kafka-shared-msk/customer-proposition/customer-proposition.tf:89]: i think we should/are supposed to prefix consumer groups with the namespace, so `customer-proposition.es-indexer-uswitch.data.v1` (and the equivalent for `uswitch.events.v1`)

**Review by mwindlinger — APPROVED** [review]

**Comment by george-angel** [comment]
We also probably want to stick to a common format for resource names best we can and use underscores as separators: https://www.terraform-best-practices.com/naming

**Pull request merged**